README.md

```diff
@@ -27,7 +27,34 @@ When developing your Lambda function locally where AWS Layer doesn't work, the D
 
 The minor version of the `datadog-lambda` package always match the layer version. E.g., datadog-lambda v0.5.0 matches the content in layer version 5.
 
-### Environment Variables
+### The Serverless Framework
+
+[The Datadog Serverless Framework Plugin](https://github.com/DataDog/serverless-plugin-datadog) should do the trick, if your Lambda function is deployed using the [Serverless Framework](https://serverless.com).
+
+Of course, you can also use the sample `serverless.yml` below as a reference, to manually include the Lambda Layer, enable tracing and set up environment variables for your Lambda function.
```

Comment:
```suggestion
Instead of the plugin, you can also use the sample `serverless.yml` below as a reference for manually including the Lambda Layer, enable AWS X-Ray tracing, and set up environment variables.
```